run.py

```diff
@@ -575,18 +513,6 @@ def run(self):
         # Set environment variables
         self.set_environment_variables()
 
-        # Initialize Phoenix instrumentation if enabled
```

Comment:
That's now done in `neuro_san_server_wrapper.py`, as it's the Neuro SAN server python process that needs to be initialized.